app/components/modals/cropper-modal.js

```diff
@@ -2,15 +2,17 @@ import ModalBase from 'open-event-frontend/components/modals/modal-base';
 
 export default ModalBase.extend({
   onVisible() {
+    let viewPort = {};
+    let factor = 150;
+    const aspectRatio = this.getWithDefault('aspectRatio', [2, 1]);
+    viewPort.width = aspectRatio[0] * factor;
```

Comment:
@sumedh123 the error is very clear. A simple google search will tell you what it means. 

Instead of
```js
    this.$('img').croppie({
      customClass : 'croppie',
      viewport    : viewport,
      boundary    : {
        height: 250
      }
    });
```

You should put 

```js
    this.$('img').croppie({
      customClass : 'croppie',
      viewport,
      boundary    : {
        height: 250
      }
    });
```

As simple as that. When you get an error, first try to understand what that error means. Please.